core/cloudflow-operator/src/main/scala/cloudflow/operator/Main.scala

```diff
@@ -87,6 +91,14 @@ object Main extends {
       """.stripMargin)
   }
 
+  private def getDeploymentOwnerReference(settings: Settings, client: skuber.api.client.KubernetesClient)(implicit ec: ExecutionContext) =
+    CloudflowOwnerReferences(
+      Await.result(client
+                     .getInNamespace[Deployment](Name.ofCloudflowOperatorDeployment, settings.podNamespace)
+                     .map(_.metadata.ownerReferences),
```

Comment:
Ah this is why it is a list.

Reply:
Another noob question: so it does not work to get the CR itself and get the metadata.uid instead and use that as an owner reference?

Reply:
Oh wait this is another owner reference, you want the owner that created the cloudflow operator..